src/ansys/dynamicreporting/core/adr_service.py

```diff
@@ -289,7 +301,7 @@ def connect(
             self.serverobj.validate()
         except Exception:
             self.logger.error("Can not validate dynamic reporting server.\n")
-            return False
+            raise NotValidServer

```

Comment:
So this is an interesting change, but the docstring does not match the change.   It does not return an error boolean, so you need a Raises section and maybe a change in the Returns section?